Multisig/Data/Services/Safe Client Gateway Service/BalancesRequest.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 struct BalancesRequest: JSONRequest {
     private let safeAddress: String
-    private let networkId: Int
+    private let networkId: String
```

Comment:
I regret that now we have multiple names interchangeably: network and chain, I will create a separate refactor ticket to align that